workers/shared/api/internal_client.py

```
@@ -431,6 +431,10 @@ def get_workflow_execution(
             response_dict = response
         return WorkflowExecutionResponse.from_api_response(response_dict)
 
+    @with_cache(
+        CacheType.WORKFLOW,
+        lambda self, workflow_id, organization_id=None: str(workflow_id),
```

Ruff (0.15.1)

[warning] Unused lambda argument: `self`

(ARG005)